### ISSUE TITLE
Can not build with bitcode enabled

### ISSUE THREAD

**Issue by ChrisLmis** [juicycleff/flutter-unity-view-widget#369]
I need to compile the app with bitcode enabled. When bitcode is enabled, I get the following build error:

ld: -U and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together

This is a result of the setup requirements for this plugin where you have to add these linker flags:

-Wl,-U,_FlutterUnityPluginOnMessage

I understand that this is required to allow Unity to talk to Flutter. Is there any way around this problem?

**Comment by juicycleff** [comment]
@ChrisLmis I'm looking for another alternative to fixing this. It's top of my todo list

**Comment by tuanha2000vn** [comment]
A temporary fix is to add these line to the end of podfile
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
    target.build_configurations.each do |config| 
      config.build_settings['ENABLE_BITCODE'] = 'NO' 
    end 
  end
end
```

**Comment by juicycleff** [comment]
This was fixed in v4.1.0, please use a matching [unitypackage](https://github.com/juicycleff/flutter-unity-view-widget/tree/master/scripts)
